static_frame/core/index_hierarchy.py

```diff
@@ -2186,12 +2196,12 @@ def _ufunc_axis_skipna(self: IH,
     def _ufunc_shape_skipna(self, *,
             axis: int,
             skipna: bool,
-            ufunc: UFunc,
-            ufunc_skipna: UFunc,
+            ufunc: TUFunc,
```

Comment:
It must be late, because it seems funny to me that people unfamiliar with this might read this as `tuff-unk`

Reply:
hahaha, tough funk!